kalite/distributed/tests/ajax_messages_test.py

```diff
@@ -0,0 +1,23 @@
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+from kalite.testing.base import KALiteBrowserTestCase
+from kalite.testing.mixins import BrowserActionMixins, FacilityMixins
+
+
+class TestShowAjaxMessages(BrowserActionMixins, FacilityMixins, KALiteBrowserTestCase):
+
+    def test_ajax_messages(self):
+
+        # Navigate to video that is not downloaded
+
+        self.browse_to(
+            self.reverse("learn") + "khan/math/algebra/introduction-to-algebra/overview_hist_alg/origins-of-algebra/")
```

Comment:
By the way, this seems to work on the Travis server, but (as I noted in one of my commit messages) there's no reason to assume that this particular video is not downloaded, in which case the test will fail. We'll need to specifically find some content item that is not available.